CHANGES.rst

```diff
@@ -7,7 +7,7 @@ Changelog
     file is managed by towncrier. You *may* edit previous change logs to
     fix problems like typo corrections or such.
     To add a new change log entry, please see
-    https://docs.pulpproject.org/en/3.0/nightly/contributing/git.html#changelog-update
+    https://docs.pulpproject.org/en/3.0/contributing/git.html#changelog-update
```

Comment:
The current ones do not contain `en/3.0/`. E.g. https://docs.pulpproject.org/contributing/git.html#changelog-update

Reply:
Similar problem with the other links you changed.

Reply:
All links updated as recommended.